internal/cmd/from_csv_flag.go

```diff
@@ -4,6 +4,7 @@ import "github.com/spf13/cobra"
 
 var fromCSVFlag string
 var csvTableNameFlag string
+var csvSeparatorFlag string
```

Comment:
Mind adding new flags on the `flags` package?
One example would be the [readonly](https://github.com/tursodatabase/turso-cli/blob/main/internal/flags/readonly.go) flag.